src/ReCaptchaLogger.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace DS\Library\ReCaptcha;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\NullLogger;
+
+/**
+ * ReCaptcha logger.
+ *
+ * @package DS\Library\ReCaptcha
+ */
```

Comment:
Codacy found an issue: [Missing @category tag in class comment](https://app.codacy.com/manual/celldweller24/ReCaptchaLibrary/pullRequest?prid=3906929)